net/flashpunk/utils/Data.as

```diff
@@ -120,11 +120,10 @@
 		}
 		
 		/**
-		 * Clears save file.
-		 * @param	filename Save file to clear.
+		 * Clears the current data.
 		 */
-		public static function clear(filename:String):void {
-			Data.load(filename);
+		public static function clear():void
+		{
 			_shared.clear();
```

Comment:
Is the desired use of `clear()` just to flush data in memory and not on disk? I feel this is the best approach, but I just want to make sure it is working as intended.

So, to delete a save file, a user would have to do the following:

``` actionscript
Data.load("slot1");
Data.clear();
Data.save("slot1");
```